pkg/selector/outputs/outputs.go

```diff
@@ -221,6 +221,8 @@ func getWideColumnsData(instanceTypes []*instancetypes.Details) []*wideColumnsDa
 				gpus = gpus + *gpuInfo.Count
 				gpuType = append(gpuType, *gpuInfo.Manufacturer+" "+*gpuInfo.Name)
 			}
+		} else {
+			gpuType = append(gpuType, none)
```

Comment:
This was added in response to this comment on another PR: https://github.com/aws/amazon-ec2-instance-selector/pull/149#discussion_r934965035